src/interpreter.ts

```diff
@@ -561,7 +567,10 @@ export class Interpreter<
     return this.send.bind(this, event);
   }
 
-  public sendTo = (event: TEvent, to: string | number | Actor) => {
+  private sendTo = (
```

Comment:
I've changed this to private - this really has seemed like undocumented and supposed to only called by the Interpreter itself, hope this is OK